### DISCUSSION TITLE
Rights assignment for annotations

### DISCUSSION THREAD

**Discussion by schultzgc** [grafana/grafana#44525] [Dashboards, Panels & Visualizations]
Currently only admins and editors can add, edit and delete annotations.
Is there a possibility to change the rights of editors or viewers in detail.
For example, that also viewers get the permission to add annotations?

**Comment by ApplicationError** [comment]
Pretty sure this issue might be related:
https://github.com/grafana/grafana/issues/9991

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
